packages/components/src/components/form/Input/Input.tsx

```diff
@@ -179,6 +178,7 @@ const Input = ({
                     rightAddonWidth={rightAddonWidth}
                     isWithLabel={!!label}
                     placeholder={placeholder || ''} // needed for uncontrolled inputs
+                    data-test={dataTest}
```

Comment:
@dahaca doesn't it make more sense to have `data-test` in the `input` itself so we are able to type on it with e2e tests?

Reply:
I am quite surprised that you need to add it here. In fact, cypress is able to type into input. but maybe playwright needs it direrctly in the input component, not in its wrapper

Reply:
Yes, it is really needed for playwright to type into it. Tested otherwise without success